app/models/conference.rb

```diff
@@ -81,6 +82,21 @@ def user_registered? user
   end
 
   ##
+  # Delete all EventSchedules that are not in the hours range
+  # After the conference has been successfully updated
+  def delete_event_schedules
+    if self.start_hour_changed? || self.end_hour_changed?
+      event_schedules = EventSchedule.select do |e|
```

Comment:
`EventSchedule.select do |e|` will go over EVERY event schedule in the database. Not cool.
We should try to scope this somehow.
Maybe adding a `has_many :event_schedules` to the conference.